fdbclient/NativeAPI.actor.cpp

```diff
@@ -1230,6 +1230,8 @@ ACTOR Future< vector< pair<KeyRange,Reference<LocationInfo>> > > getKeyRangeLoca
 	}
 }
 
+// Get the SS locations for each shard in the 'keys' key-range;
+// Returned vector size is the number of shards in the input keys key-range.
```

Comment:
It's worth mentioning that the first part of each pair is actually the whole shard key-range, not a part of the given key range. So they are `<ShardRange, storage server location info>` pairs. 

Concretely, if you query the function with key range `(b, d)`, the returned list of pairs could be something like:

```
[((a, b1), locationInfo), ((b1, c), locationInfo), ((c, d1), locationInfo)]
```
I added a line for this function on master branch.